skywalking-collector/skywalking-collector-worker/src/main/java/com/a/eye/skywalking/collector/worker/receiver/TraceSegmentReceiver.java

```diff
@@ -0,0 +1,46 @@
+package com.a.eye.skywalking.collector.worker.receiver;
+
+import com.a.eye.skywalking.collector.actor.AbstractMember;
+import com.a.eye.skywalking.collector.actor.AbstractWorker;
+import com.a.eye.skywalking.collector.actor.AbstractWorkerProvider;
+import com.a.eye.skywalking.collector.worker.WorkerConfig;
+import com.a.eye.skywalking.collector.worker.application.ApplicationMember;
+import com.a.eye.skywalking.collector.worker.applicationref.ApplicationRefMember;
+import com.a.eye.skywalking.trace.TraceSegment;
+
+/**
+ * @author pengys5
+ */
+public class TraceSegmentReceiver extends AbstractWorker {
+
+    @Override
+    public void preStart() throws Exception {
+        ApplicationMember.Factory factory = new ApplicationMember.Factory();
+        factory.createWorker(memberContext(), getSelf());
+    }
+
+    @Override
+    public void receive(Object message) throws Throwable {
+        if (message instanceof TraceSegment) {
+            TraceSegment traceSegment = (TraceSegment) message;
+
+            AbstractMember applicationMember = memberContext().memberFor(ApplicationMember.class.getSimpleName());
```

Comment:
* From my understanding, `memberFor` is for finding a member, so I prefer `find`.
* `find` method's parameter should stay in `Class<?>`, it's a better style.